### DISCUSSION TITLE
Misleading data on deployment table (Might be related to queuing bug which I am also experiencing)

### DISCUSSION THREAD

**Discussion by amrhassab** [vercel/community#4434] [Bug Report]
### Page to Investigate

https://vercel.com/amrhassabs-projects/my-portfolio/deployments

### Steps to Reproduce

Steps to reproduce located below images

issue: The deployment table shows redeploying 5f9c2l8d6 (image 1) when clicking on it you see that it is actually 6b20f09 (image 2) - the first image is showing the wrong data 

image 1
![image 1](https://github.com/vercel/community/assets/15644210/4737309d-0699-423d-8a98-aa1afd1fcfbe)

image 2
![image 2](https://github.com/vercel/community/assets/15644210/730bc014-9d91-485b-a871-48812982e1c3)

Steps

1) in a new project deploy a commit that fails in the build phase - let's call this commit [5f9c2l8d6]
2) make a new commit that fixes the issues (push didn't build but that's a different issue) - let's call this commit [6b20f09]
3) delete the failed deployment from your table 
4) delete the project 
6) add the project again and deploy that successfully           
7) go to Speed Insights and enable - it will prompt you to redeploy - press redeploy
8) you will see that on top of unnecessary queuing (since there is only one build) the data displayed on the table is not correct

**Comment by amyegan** [comment]
Hi @amrhassab! The [build processing incident](https://www.vercel-status.com/incidents/6zzfgq5m2jqs) is resolved now. Do you still see incorrect deployment data on new builds?

**Comment by amrhassab** [comment]
Im not sure I deleted that deployment but deployments are working much better now @amyegan - thanks!
